### DISCUSSION TITLE
about quectel board

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#317] [General]
hi @herlesupreeth  are you tested lte or 5g in quectel board

**Comment by herlesupreeth** [comment]
Yes, i have Quectel RM520N-GL and it works with both 4G and 5G SA. I am not sure whether it supports VoLTE or VoNR

**Reply by Sivanesh1992** [reply]
@herlesupreeth  thankyou . i am using AG521REU to test LTE but i am able to register network but after few seconds its deregister please help me resolve the issue i will share pcap .

i want to know open 5gs support ipv4v6 or only ipv4 ?

i am not able to set apn for ipv4 getting below error.

:] AT+Cgdcont=1,“IPv4”,“internet”,“0.0.0.0”,0,0,0,0
:] AT+Cgdcont=1,“IPv4”,“internet”,“0.0.0.0”,0,0,0,0
:] ERROR

**Comment by herlesupreeth** [comment]
> i am using AG521REU to test LTE but i am able to register network but after few seconds its deregister please help me resolve the issue i will share pcap .

could be due to UE inactivity. But does the UE reattach when you try a ping to UE IP from UPF?

> i want to know open 5gs support ipv4v6 or only ipv4 ?

I think open5gs is able to provide ipv4v6 and ipv6 only. I havent tested ipv4v6 so cant say whether it works perfectly or not. But ipv6 only works well

> 
> :] AT+Cgdcont=1,“IPv4”,“internet”,“0.0.0.0”,0,0,0,0
> :] AT+Cgdcont=1,“IPv4”,“internet”,“0.0.0.0”,0,0,0,0
> :] ERROR

I would refer to the AT command manual for your modem. I am not sure whats the syntax for it. I normally use modemmanager from linux to initiate a PDN connection to an APN.

`sudo mmcli -m 0 --simple-connect="apn=internet"`

**Reply by Sivanesh1992** [reply]
thankyou @herlesupreeth
